3.process_cfret_features/scripts/1.sc_quality_control.py

```diff
@@ -73,55 +90,37 @@
 plate_df.head()
 
 
-# ## Identify mis-segmented nuclei from large clusters
+# ## Oversegmented nuclei with very high intensity (clusters)
```

Comment:
Hmm, interesting point. I think it is fine to leave oversegmented since it better explains what this condition is looking for when identifying technical outliers. Thank you for pointing it out though!